### DISCUSSION TITLE
Rust

### DISCUSSION THREAD

**Discussion by gngrossi** [zopencommunity/meta#280] [General]
Any plans for a Rust language port?
thanks

**Comment by melanieullmer** [comment]
Good question.  Do you have a specific use case in mind?

**Reply by gngrossi** [reply]
Currently, we don't. I wanted to inquire about Rust and whether it would begin as a port via this channel and later become available like Node.js and Go. Thoughts?

**Reply by v1gnesh** [reply]
There's a poll - please vote for it!

https://github.com/orgs/ZOSOpenTools/discussions/298

zOS can level up quite a bit by having Rust (and its ecosystem).
